scenes/game_elements/props/decoration/hiding_mushroom/hiding_mushroom.tscn

```diff
@@ -140,5 +142,13 @@ collision_layer = 4
 [node name="CollisionShape2D" type="CollisionShape2D" parent="PlayerDetector"]
 shape = SubResource("CircleShape2D_v208a")
 
+[node name="AudioStreamPlayer2D_Hide" type="AudioStreamPlayer2D" parent="."]
+stream = ExtResource("5_gyonr")
+volume_db = 2.0
+
+[node name="AudioStreamPlayer2D_Reveal" type="AudioStreamPlayer2D" parent="."]
```

Comment:
I don't think you need the type of the node in its name. I think you could rename these to `HideSound` and `RevealSound` (and update the code accordingly).